mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -235,6 +235,13 @@ region are guaranteed to support the same version.
     manages the default node pool, which isn't recommended to be used with
     Terraform. Structure is documented below.
 
+* `network_config` -  (Optional, [Beta](https://terraform.io/docs/providers/google/guides/provider_versions.html)) Configuration for
+   [Adding Pod IP address ranges](https://cloud.google.com/kubernetes-engine/docs/how-to/multi-pod-cidr))  used in creating the default node pool.
+    Generally, this field should not be used at the same time as a
```

Comment:
This looks like it was copied from the field above?

Reply:
Fixed